packages/theme/src/theme/_forms.scss

```diff
@@ -6,6 +6,11 @@
 $form-shadow: inset 0 -1px 0 $gallery;
 $form-focus-shadow: inset 0 -2px 0 $scooter;
 
+@mixin checkOutline() {
```

Comment:
The linter complained about camelCase, but while at it, why not checkbox-outline?

Reply:
Done